src/style.css

```diff
@@ -0,0 +1,116 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.product-card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 15px;
+}
+
+.product-card__image {
+  content: url(images/imac.jpeg);
+  width: 160px;
+  height: 134px;
+  margin: 30px 0;
+}
+
+.product-card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  width: 170px;
+  margin: 15px 0 0;
+}
+
+.product-card__code {
+  width: 170px;
+  font-size: 11px;
+  line-height: 13px;
+  margin: 5px 0;
+}
+
+.product-card__rating {
+  display: flex;
+  width: 170px;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: top;
+  margin-top: 10px;
+}
+
+.product-card__stars * {
+  display: inline;
+  padding: 1px;
+  content: url(images/star.svg);
+
+}
+
+/* .product-card__stars > *, */
+.product-card__stars :nth-child(-n+4),
+.product-card__stars :nth-child(-n+3),
+.product-card__stars :nth-child(-n+2),
+.product-card__stars :first-child {
+  content: url(images/star-active.svg);
+}
```

Comment:
.product-card__stars should have modifier `--n` which means how many stars should be active. Now all those rules works together, and you ever have 4 active stars